src/containers/CreateDoc/index.tsx

```diff
@@ -2,8 +2,28 @@ import React from 'react';
 import Dropdown from '../../components/Dropdown';
 import documentTypes from './config';
 import { createPdf } from './pdf';
+import { connect } from 'react-redux'
+import { setDocType } from './createDocSlice';
 
-const CreateDoc: React.FC = () => {
+interface CreateDocProps {
+    setDocType: (value: string) => void,
+}
+
+const getDocumentType = (id: Number) => {
+    const type = documentTypes.filter(type => type.id === id);
+    if (!type || !type.length) return '';
+    return type[0].value;
+}
+
+const CreateDoc: React.FC<CreateDocProps> = ({ setDocType }) => {
+
+    const [dropdownValue, setDropdownValue] = React.useState<string | number>();
```

Comment:
I disagree, this should NOT create an state of it's own. This should watch whats on the redux state to bring the document type.